src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+function ButtonPanel() {
+  return (
+    <div className="center-calculator">
+      <div className="board-row">
+        <button type="button" className="square">AC</button>
+        <button type="button" className="square">+/-</button>
+        <button type="button" className="square">%</button>
+        <button type="button" className="square">÷</button>
+      </div>
+      <div className="board-row">
+        <button type="button" className="square">7</button>
+        <button type="button" className="square">8</button>
+        <button type="button" className="square">9</button>
+        <button type="button" className="square">X</button>
+      </div>
+      <div className="board-row">
+        <button type="button" className="square">4</button>
```

Comment:
- [ ] Please import the Button component here and rather than using a regular HTML button simply use the Button component and pass the name prop to it.